go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/graphql-go/graphql v0.7.8 // indirect
 	github.com/hashicorp/golang-lru v0.5.1 // indirect
 	github.com/ianlancetaylor/demangle v0.0.0-20181102032728-5e5cf60278f6 // indirect
+	github.com/inconshreveable/mousetrap v1.0.0 // indirect
```

Comment:
is this needed?

Reply:
No idea, I didn't add it manually. I can guess these files are dependencies, but honestly I didn't even read about them.